src/ui/prompts/variablePrompts.ts

```diff
@@ -132,13 +131,13 @@ export const getVariableValuePrompt = (
 ) => {
     switch (variableType) {
         case 'Boolean':
-            return variableValueBooleanPrompt(variation.key, defaultValue as boolean | undefined)
+            return variableValueBooleanPrompt(variation.key, defaultValue as boolean | undefined, variation.name)
```

Comment:
might be more clear to just pass the full variation as a param?

Reply:
since these methods are also used by `variations update` and the cleanup command, i had to use discriminated unions to differentiate when to access `name`. It's a little more clean than passing separate params